src/player_event_handler.rs

```diff
@@ -0,0 +1,50 @@
+use std::process::Command;
+use std::sync::mpsc::{channel, Sender};
+use std::thread;
+use librespot::playback::config::PlayerEvent;
+
+fn run_program(program: &str, args: Vec<String>) {
+    info!("Running {}", program);
+    let mut v: Vec<&str> = program.split_whitespace().collect();
+    let status = Command::new(&v.remove(0))
+        .args(&v)
+        .args(args)
```

Comment:
I think it's better to use environment variables here so the values have names associated. Additionally include a `PLAYER_EVENT=start` etc.. variable so a single program can deal with all events.

You can use a HashMap to pass the arguments in to this function.